src/Tests/MPQEditor/MPQData.test.ts

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { assert } from "chai";
+import { MPQData } from "../../MPQEditor/MPQData";
+
+const defaultMPQText = `{
+ "default_quantization_dtype": "uint8",
+ "default_granularity": "channel",
+ "layers": [],
+ "model_path": "sample.circle"
```

Comment:
where can I find `sample.circle` ?

Reply:
`sample.circle` is supposed to be located in the same directory as the `.mpq.json` file.

Reply:
It's a bit uncertain about this. I cannot find `sample.circle` in draft PR #1505 and cannot find any code that uses `model_path` key. 
What's the use of this?

Reply:
@seanshpark 
Please see https://github.com/Samsung/ONE-vscode/pull/1505/files#r1147197657

Reply:
ok thanks, got it.